ci-operator/step-registry/baremetal/lab/pre/dhcp-conf/baremetal-lab-pre-dhcp-conf-ref.yaml

```diff
@@ -16,6 +16,8 @@ ref:
       default: "true"
     - name: INTERNAL_NET_IP
       default: "192.168.80.1"
+    - name: IPV6_ENABLE
```

Comment:
@jadhaj i've seen the agent jobs have ipv4_enable and ipv6_enable as lower-case variables. Could we stay all in sync (i'd say use lowercase here too to avoid changing the working ones)

Reply:
yes, sure